e2e/volume_ops_test.go

```diff
@@ -261,6 +261,53 @@ func testVolumeExpand(t *testing.T, tc *testCluster) {
 	for _, subvol := range volinfo.Subvols {
 		r.Len(subvol.Bricks, 2)
 	}
+
+	createReqBrick := api.VolCreateReq{
+		Name: "TestVolumeExpand",
+		Subvols: []api.SubvolReq{
+			{
+				ReplicaCount: 2,
+				Type:         "replicate",
+				Bricks: []api.BrickReq{
+					{PeerID: tc.gds[0].PeerID(), Path: brickPaths[4]},
+					{PeerID: tc.gds[1].PeerID(), Path: brickPaths[5]},
+				},
+			},
+			{
+				Type:         "replicate",
+				ReplicaCount: 2,
+				Bricks: []api.BrickReq{
+					{PeerID: tc.gds[0].PeerID(), Path: brickPaths[6]},
+					{PeerID: tc.gds[1].PeerID(), Path: brickPaths[7]},
+				},
+			},
+		},
+		Flags: flags,
+	}
+
+	_, err = client.VolumeCreate(createReqBrick)
+	r.Nil(err)
+
+	expandReq = api.VolExpandReq{
+		ReplicaCount: 3,
+		Bricks: []api.BrickReq{
+			{PeerID: tc.gds[0].PeerID(), Path: brickPaths[8]},
+			{PeerID: tc.gds[1].PeerID(), Path: brickPaths[9]},
+		},
+		Flags: flags,
+	}
+
+	//expand with new brick dir which is not created
+	volinfo, err = client.VolumeExpand("TestVolumeExpand", expandReq)
```

Comment:
@vpandey-RH can you add some negative test scenarios also.

Reply:
Sure

Reply:
@Madhu-1 I have added a couple of negative cases.